### PR TITLE
Change OCW Next build to use new repos

### PR DESCRIPTION
**Pull request by markbreedlove**
This PR revises the provisioning and configuration of the OCW Next build server to accommodate the new build components.

I've been running with the enclosed changes on the RC build server.

### CODE REVIEW

File: pillar/apps/ocw-next-production.sls [hunk 1]
```diff
@@ -4,9 +4,11 @@ ocw-next:
   source_data_bucket: open-learning-course-data-production
   search_api_url: //open.mit.edu/api/v0/search/
   ocw_to_hugo_git_ref: release
-  hugo_course_publisher_git_ref: release
+  ocw_www_git_ref: main
```
Comment (blarghmatey) [inline pillar/apps/ocw-next-production.sls:7]: Are we going to deploy QA and production from the same branch?

Comment (markbreedlove) [reply pillar/apps/ocw-next-production.sls:7]: Yes, but only because at this moment there is one branch. I expect this to change.

**Comment by blarghmatey** [comment]
:+1:

File: pillar/apps/ocw-next-production.sls (hunk 1 above)
Comment (markbreedlove) [reply pillar/apps/ocw-next-production.sls]: I've changed to `release` for production in https://github.com/mitodl/salt-ops/pull/1336/commits/4a219844904f93fa3500b4136d1efba6161ecae6 and will change RC later to `release-candidate` when such a branch exists.

**Pull request merged**